cirq-ionq/cirq_ionq/service_test.py

```diff
@@ -159,22 +159,53 @@ def test_service_list_calibrations():
 
 
 def test_service_api_key_via_env():
-    os.environ['IONQ_API_KEY'] = 'tomyheart'
-    service = ionq.Service(remote_host='http://example.com')
-    assert service.api_key == 'tomyheart'
-    del os.environ['IONQ_API_KEY']
+    with mock.patch.dict('os.environ', {'IONQ_API_KEY': 'tomyheart'}):
```

Comment:
Please pass `os.environ` as is instead of as a string.
For these tests you can consider decorating the test method, for example,
```python
@mock.patch.dict(os.environ, {'IONQ_API_KEY': 'tomyheart'})
def test_service_api_key_via_env():
    service = ionq.Service(remote_host='http://example.com')
    ... 
```
Note if needed, you can also modify `os.environ` in the test function body.  Any changes will be reverted when function returns and exits the `patch.dict` context.